db/migrations/core-data/20240417115737.do.school-pin-data.js

```diff
@@ -0,0 +1,884 @@
+'use strict'
+
+module.exports.generateSql = function () {
+
+  if (process.env.ALLOWED_WORDS_SET !== undefined) {
+    const sql = process.env.ALLOWED_WORDS_SET.split(',').map(word => {
+      return `INSERT INTO mtc_admin.[schoolPin] (val) VALUES ('${word}')`
```

Comment:
I would personally like to see whitespace trimmed and the word lowercased to enforce correct behaviour.